### PR TITLE
Fix tab name in product form

### PR DESCRIPTION
**Pull request by zeroheure**
Tab name is "Inventory" instead of "Purchase"

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Well, there are multiple paths for checking the products, so maybe the explanation would be larger specifying that it would be available if module X is installed.

**Comment by zeroheure** [comment]
Added

**Review by pedrobaeza — APPROVED** [review]

**Comment by zeroheure** [comment]
Travis failed for external reasons : unable to downdload Python. How can I relaunch it ?

**Comment by pedrobaeza** [comment]
Relaunched

**Comment by zeroheure** [comment]
ping

**Review by carlosdauden — APPROVED** [review]

File: product_supplierinfo_revision/README.rst [hunk 1]
```diff
@@ -23,8 +30,9 @@ To use this module:
 #. By clicking on Apply button, a new supplier pricelist will be created with
    a price increased accordingly to the percent entered. Old supplier
    pricelist will have a date end from which pricelist will be out of date.
-#. You can check supplier pricelists as well as variation percent in the
-   product form, by clicking on *Purchase* tab.
+#. Once the Inventory Management (*stock*) module is installed, you can check 
+   supplier pricelists as well as variation percent in the product form, by 
+   clicking on *Inventory* tab.
```
Comment (carlosdauden) [inline product_supplierinfo_revision/README.rst:35]: On client instance I look suppliers in Purchase tab, but if you has check...

Comment (zeroheure) [reply product_supplierinfo_revision/README.rst:35]: I don't remember exactly but I think that the location depends on installed modules. 
I've checked again and if Purchase Management is install, suppliers are located in product's Purchase tab. I will look further, don't merge.

**Pull request merged**